DirectProgramming/DPC++FPGA/Tutorials/Features/latency_control/src/latency_control.cpp

```diff
@@ -0,0 +1,123 @@
+//==============================================================
```

Comment:
You can remove this header comment

Reply:
Will do. Why we don't need it now?

Reply:
It seems that newer samples got rid of it, as the license is written in so many places already...
At least it removes "bloat" from the actual code we expect the users to read

Reply:
Good to know!